Revert "Remove `ArticleDesign.PrintShop`" (#1720)
guardian/csnx: .changeset/small-bats-help.md

---

> "@fake-scope/fake-pkg": major

Reviewer: i think this would be a minor right? and then we can deprecate 19.0.0?

> "@fake-scope/fake-pkg": major

Reviewer: i think this would be a minor right? and then we can deprecate v19.0.0?

```suggestion
"@fake-scope/fake-pkg": minor
```

> "@fake-scope/fake-pkg": minor

Reviewer 2: Unless I'm missing something this should be `@guardian/libs`?

```suggestion
"@guardian/libs": minor
```

> '@fake-scope/fake-pkg': minor

Reviewer: is this right?
```suggestion
'@fake-scope/fake-pkg': minor
```

> '@fake-scope/fake-pkg': minor

Reviewer: ```suggestion
'@guardian/libs': minor
```

Author: Sorry, I charged ahead with the automated changeset without looking closely enough 🙈

---

> Revert "Remove `ArticleDesign.PrintShop`". To be re-implemented at a later date.

Reviewer: ```suggestion
Restores `ArticleDesign.PrintShop`. To be re-removed at a later date.
```
